python/chronos/src/bigdl/chronos/benchmark/utils_env.py

```diff
@@ -116,8 +110,8 @@ def get_nano_env_var(use_malloc: str = "tc", use_openmp: bool = True,
     if jemalloc_lib_dir is not None:
```

Comment:
there is a KMP_AFFINITY check, please change it to `"granularity=fine"` since we are currently use this.

Reply:
has revised